windows-installer/stackdriver_logging_agent.nsi

```diff
@@ -0,0 +1,311 @@
+;--------------------------------
```

Comment:
I'd rather be more specific.  How would you feel about renaming the monitoring agent nsi file instead?